### DISCUSSION TITLE
502 Bad Gateway

### DISCUSSION THREAD

**Discussion by lehvairo** [eduardogsilva/routerfleet#39] [General]
![image](https://github.com/user-attachments/assets/5f8b9141-fce2-4793-aae6-78a6a73f57e3)

**Comment by eduardogsilva** [comment]
Hello @lehvairo! Sorry for taking so long to answer, but probably one of your containers wasn't running. (Maybe postgres or even the main application).

If the issue still persists, please take a look on your docker logs.

**Comment by lehvairo** [comment]
Here is the .env file. I can't find the docker.log file. Whats the location?

![Kuvatõmmis 2024-08-28 095014](https://github.com/user-attachments/assets/1a2ac290-bc3d-47bf-a224-eaab21866b49)

**Comment by eduardogsilva** [comment]
I can see that your browser points to https://10.166.200.3 but your env file shows 10.166.200.5.

You can check if your containers are running. 
Another test that you can try is to run docker compose up without the -d flag and take a closer look on the logs.

Cheers

**Comment by lehvairo** [comment]
I have changed browser point 10.166.200.5 in env file. 

![Kuvatõmmis 2024-08-29 082809](https://github.com/user-attachments/assets/75316c59-9106-4638-b083-22c69f5738bf)

More
![Kuvatõmmis 2024-08-29 141259](https://github.com/user-attachments/assets/0817ce19-2b9d-4bec-9961-040799ba1b8c)

**Comment by eduardogsilva** [comment]
Looks like your .env is not properly set.

I took a closer look on your .env file and I noticed that you are editing the file conf.env but it should be ".env" only.

![image](https://github.com/user-attachments/assets/bc3b01bc-b9f8-4402-a70c-aeacf2b31826)
